netkan/netkan/cli.py

```diff
@@ -105,18 +105,18 @@ def indexer(queue, metadata, token, repo, user, key,
     '--schedule-all', is_flag=True, default=False,
     help='Schedule all modules even if we think they should be covered by webhooks'
 )
-def scheduler(queue, netkan, max_queued, debug, dev, schedule_all):
+def scheduler(queue, netkan_remote, max_queued, debug, dev, schedule_all):
     level = logging.DEBUG if debug else logging.INFO
     logging.basicConfig(
         format='[%(asctime)s] [%(levelname)-8s] %(message)s', level=level
     )
     logging.info('Scheduler started at log level %s', level)
 
-    scheduler = NetkanScheduler(Path('/tmp/NetKAN'), queue, force_all=schedule_all)
-    if scheduler.can_schedule(max_queued, dev):
-        init_repo(netkan, '/tmp/NetKAN')
-        scheduler.schedule_all_netkans()
-        logging.info("NetKANs submitted to {}".format(queue))
+    sched = NetkanScheduler(Path('/tmp/NetKAN'), queue, force_all=schedule_all)
+    if sched.can_schedule(max_queued, dev):
```

Comment:
What was the motivation for shortening `schedule`?

Reply:
Fixing this:

![image](https://user-images.githubusercontent.com/1559108/67298526-fe564980-f4b0-11e9-82c9-f7631bc79c1c.png)

Reply:
It's redefining the function name, which as it's the cli it's in the global (outer) scope. I'll wear that as a facepalm on my part :man_facepalming: